client/.env.sample

```diff
@@ -8,4 +8,5 @@ REACT_APP_OPENSRP_CLIENT_ID=test1
 REACT_APP_ENABLE_OPENSRP_OAUTH=true
 REACT_APP_OPENSRP_API_BASE_URL=https://reveal-stage.smartregister.org/opensrp/rest/
 REACT_APP_OPENSRP_USER_URL=https://reveal-stage.smartregister.org/opensrp/user-details
-REACT_APP_DISABLE_LOGIN_PROTECTION=false
\ No newline at end of file
+REACT_APP_DISABLE_LOGIN_PROTECTION=false
+REACT_APP_SENTRY_DSN=https://something@sentry.onalabs.org/12345
```

Comment:
lets document this [here](https://github.com/OpenSRP/web/blob/master/docs/env.md) as well.